nncf/config_schema.py

```diff
@@ -306,7 +341,7 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
             "patternProperties": {
                 ".*": {
                     "type": "object",
-                    "properties": QUANTIZER_CONFIG_PROPERTIES,
+                    "properties": QUANTIZER_CONFIG_INIT_PROPERTIES,
```

Comment:
Suggest to do the following:
```
"properties": {
**QUANTIZER_CONFIG_PROPERTIES,
**RANGE_INIT_CONFIG_PROPERTIES,
}
```

Might also want to do the same in `QUANTIZER_GROUP_SCHEMA` - so that it is possible to set different inits for entire activation or weight quantizer sets.

Reply:
Done. Might do separate initialization configs for quantizer groups in a separate PR.